### ISSUE TITLE
In Bot telemetry the user message is not added by default with the current bot telemetry custom dimension

### ISSUE THREAD

**Issue by SSanjeevi** [microsoft/BotFramework-Composer#8497]
## Describe the bug

In Bot telemetry the user message is not added by default with the current bot telemetry custom dimensions

## Version

2.1.0

## Browser

- Electron distribution

## OS

Windows OS

## To Reproduce

Follow this document to enable Bot telemetry in app insights.
https://docs.microsoft.com/en-us/composer/how-to-capture-telemetry

After successful enabling of bot telemetry, try to see the logs of custom event in application insights and found the user text entered on each message is not added in the custom dimension especially on intent Luis match events, this make difficult to get visual of what is the user message and which utterance matched based on that.
Can you please guide me to enable that, i don't see them configurable anywhere?

i don't especially see "Text in message" missing in logs under custom dimension text field.

Also see the bot response with the syntax of the code not the actual output adaptive card or text reply.
something like this: 
"#SendActivity_GreetingReturningUser"

"Not the inner activity derived output"

## Expected behavior

Custome dimensions should contain user message in logs under text field.

**Comment by anusharr** [comment]
@SSanjeevi, thanks for reporting this. Let me investigate further and update the thread accordingly.

**Comment by SSanjeevi** [comment]
@anusharr - after enabling the personal information the contents are getting updated in app insights logs.
Only thing is sometimes it's not logging the complete response instead the activity name bot response : "#SendActivity_GreetingReturningUser"

**Comment by anusharr** [comment]
@SSanjeevi, so is the content getting updated if you change the `logPersonalInformation` to `true`?

**Comment by SSanjeevi** [comment]
Yes it's getting updated but with actitvity name not the response adaptive card. Is that something we can do?

**Comment by benbrown** [comment]
@SSanjeevi does the bot respond as expected? 

Can you paste the LG content from this dialog so we can take a look? There may be a syntax error or something that is causing it to pull in this value rather than the actual text.

**Comment by SSanjeevi** [comment]
@benbrown we are able to get the same cards as response from bot, only thing is it's logging the response in logs telemetry.
Even some messages are missing.
